### DISCUSSION TITLE
How many sumolings require Developer plan + Storage + Transcription + Remote Access? (Question from Founders Team)

### DISCUSSION THREAD

**Discussion by fareed069417** [v-empower/MeetHour-Web-MobileSDKs#72] [Polls]
Hey Sumolings!

We’re exploring interest in a Developer Tier 4 for Meet Hour, tailored specifically for API and SDK access. We’ve noticed a growing demand for a developer-focused option, so we're considering launching a new lifetime deal tier.

Tier 4 will be consisting of all the following features.

1.⁠ ⁠All Pro Features
2.⁠ ⁠⁠Remove 100% Meet Hour Branding
3.⁠ ⁠⁠200GB storage
4. APIs & SDKs access with 10 Developer Plan Licences
5.⁠ ⁠⁠Auto Start Recording
6.⁠ ⁠⁠Auto Start Live Streaming
7.⁠ ⁠⁠Custom Domain
8.⁠ ⁠⁠Branded Conference 
9. 1080p recording
10. 10 Developer Licences

**Future Enhancements**
11.⁠ ⁠Breakout Rooms
12.⁠ ⁠⁠Multi Lingual Application
13.⁠ ⁠⁠⁠Integrations with Pabbly and Zapier.
14.⁠ ⁠⁠Annotations on screen sharing
15.⁠ ⁠⁠⁠Transcriptions
16. Remote Control 

Existing Appsumo Deal - https://appsumo.com/products/meet-hour/ 

Costing for Developer Plan - USD 999 for Life Time Deal. 
Costing for Branded Apps - USD 399

Offer page - https://meethour.io/products/video-conference/black-friday-developer

To make this tier a reality, we’re aiming for at least 50 interested Sumolings! If this sounds like something you’d benefit from, let us know by confirming your interest!

For more details, please visit the Developer Plan - https://meethour.io/products/video-conference/developer 

<img width="1622" alt="Screenshot 2024-10-27 at 12 32 37 AM" src="https://github.com/user-attachments/assets/cbd05d35-e822-4b26-89cc-c2b40888a95f">

**Comment by darylcrouse** [comment]
yes, i would like the developer tier

**Reply by fareed069417** [reply]
@darylcrouse - Thanks for your interest. Hope you have voted above.

**Reply by selcukca** [reply]
I am interested in deveoloper tier. How can  I purchase it?

**Reply by fareed069417** [reply]
I would request you to get in touch with our customer care related to this.

**Reply by fareed069417** [reply]
Here is the Payment Link - https://bit.ly/4e4dXG8

**Reply by prodos** [reply]
Greetings. I have voted "I am interested".

Question: The current AppSumo Tier 3 plan offers "15 host license(s)" -- which means that there can be 15 conferences happening at the same time. 

How does this compare with the Developer Plan LTD?

Thanks for any clarification.

Best Wishes,
Prodos

**Reply by fareed069417** [reply]
@prodos - In this plan, we are offering 10 Developer licences in this plan. So, you can run upto 10 parallel conferences with integration capabilities.

**Reply by prodos** [reply]
Thanks for that information, Fareed.
:-)
Best Wishes,
Prodos

**Reply by prodos** [reply]
Hello again Fareed,

The Developer Plan offers: "Remove 100% Meet Hour Branding".
Does this differ from the white labelling that is offered in the AppSumo Tier 3 LTD?
Thanks for any further clarification.
:-)
Best Wishes,
Prodos

**Reply by fareed069417** [reply]
It is same like Tier 3 "Remove 100% Meet Hour Branding" in conference, branded domains and even recording.

**Reply by prodos** [reply]
Thanks for that information, Fareed.
:-)
Best Wishes,
Prodos

**Reply by spyrobinson** [reply]
I have Tier 3, when will Tier 4 will be available? I stumbled upon this potential upgrade in the comments section, so many Appsumo users will not know about this voting so you may not get 50 votes quickly.

**Reply by fareed069417** [reply]
@spyrobinson - We started selling Developer Plan Life Time deal already on our website. You can check it out - https://meethour.io/products/video-conference/black-friday-developer

**Comment by darylcrouse** [comment]
I already have tier 3 so please make sure that it's not a new deal otherwise I won't be able to use the upgrade credit for what I've alredy purchased. Otherwise, please set it up ASAP so I can refund the current tier 3 and buy the new deal.

**Reply by fareed069417** [reply]
@darylcrouse - Sure. We are waiting for other people to show interest. Appsumo people has to agree for that. Moreover, if they don't agree we are planning to sell this on our website.

**Reply by fareed069417** [reply]
Here is the Payment Link - https://bit.ly/4e4dXG8

**Reply by webstudio2024** [reply]
Had I not read deeper in the comments section, i wouldn't have found out about this page. You should have put this up in the founders' update section.

**Reply by fareed069417** [reply]
@webstudio2024 - Would you be interested in Developer Plan?

**Reply by webstudio2024** [reply]
> @webstudio2024 - Would you be interested in Developer Plan?

Hi fareed! I am and I have placed my vote in the poll because you are gathering interested parties. However, i do have my opinions.

At the current price tag and with that limits, I am not really in favor. Especially when this is a private deal that you're currently offering that does not even come close to the guarantee that AppSumo provides for their Plus members.

I got the Tier 3, and I am still considering if I want to hold on to it or not. As you know BF is round the corner, so if you want to launch the Tier 4 in AppSumo, i would suggest doing it sooner rather than later because Sumolings like myself would definitely need time to weigh our options and we only have so much to spend during this time of the season and not forgetting there are many other tools that will be dropping as well.

Nevertheless, if you still want to gather votes before launching the Tier 4 in AppSumo, then you should really put up an update in the founders section as soon as possible. At least there will be more people reading it.

Good luck with the campaign!

**Reply by fareed069417** [reply]
@webstudio2024 - Thanks for your input, Appsumo company may not agree to make changes in the deal at the moment, they would rather sell it as a different deal. As so many people are showing interest. We are thinking of selling it as a private deal on our website..

**Comment by fareed069417** [comment]
Guys, do you want branded apps as well along with Developer plan? We are offering Branded apps (Android & iOS Apps) along with Developer plan at an additional charge of $399. Check out details on this page -https://meethour.io/products/video-conference/features/branded-conference

Payment link for Apps - https://meethour.io/products/video-conference/black-friday-developer

**Comment by fareed069417** [comment]
We have released developer plan life time deal in our blackfriday offer. Check it out - https://meethour.io/products/video-conference/black-friday-developer

**Comment by spyrobinson** [comment]
Hi Shoeb,

Thank you for the update. I'm interested in learning more about the
Developer Plan Lifetime deal.

Could you please clarify what is included in the 10 developer licenses?
Specifically, does this mean we can use 10 custom domains etc.?

Additionally, if we purchase the branded apps option as well, does that
apply to each of the 10 developer licenses seperately?

Thanks for your assistance.

Best,
Ian Robinson

**Reply by fareed069417** [reply]
@spyrobinson - The Developer Plan is an enhanced version of the Pro Plan, designed for advanced use cases with API and SDK integration capabilities. It includes a 10-developer license, allowing you to host 10 parallel calls across up to 5 integrated websites. You can connect up to 5 domains for this purpose. Moreover, you can upto 10 host users as well.

BlackFriday Life Time Developer Licence @ $999 - https://meethour.io/products/video-conference/black-friday-developer

However, the Branded Domains SaaS feature is limited to one custom domain only. Additionally, branded apps (for Android and iOS) purchased separately will be exclusively linked to this one custom branded domain. For more details, visit: [Branded Conference Features](https://meethour.io/products/video-conference/features/branded-conference).

**Comment by fareed069417** [comment]
We are amazed with the response we are getting in Developer Plan. Hurry don't miss the black friday offer. - https://meethour.io/products/video-conference/black-friday-developer

**Comment by fareed069417** [comment]
Guys, our Developer plan Life Time Deal offer is about to end. Only few hours left. Hurry Don't miss this offer - https://meethour.io/products/video-conference/black-friday-developer

You may regret it if you miss this.

**Comment by fareed069417** [comment]
Meet Hour Life time deal is back - https://meethour.io/products/video-conference/black-friday-developer
